docs/application/toc_all.md

```diff
@@ -16,6 +16,13 @@
 ## [Overview](/application/dotnet/index.md)
 ## [Learn about .NET](/application/dotnet/learn-about-dotnet.md)
 
+## Get Started using VSCode
+### [.NET application development](/application/vscode-ext/Tizen/dotnet.md)
+
+## Get Started using VS
```

Comment:
Hello. I have one more comment. 

\## Get Started using VS

\## Get Started using VSCode

VS and VSCode.

I think we'd better write as "Visual Studio" and "Visual Studio Code Extension"